https://satellity.org/,Satellity,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: URLError(gaierror(8, 'nodename nor servname provided, or not known')) | reason: [Errno 8] nodename nor servname provided, or not known
https://dev.guardianproject.info/projects/ostel/wiki/Server_Documentation,Ostel,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: URLError(ConnectionRefusedError(61, 'Connection refused')) | reason: [Errno 61] Connection refused
https://gosora-project.com/,Gosora,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: <HTTPError 500: 'Internal Server Error'> | reason: Internal Server Error
https://www.drupal.org/,Drupal,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: <HTTPError 403: 'Forbidden'> | reason: Forbidden
https://reactioncommerce.com/,Reaction Commerce,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: URLError(ConnectionRefusedError(61, 'Connection refused')) | reason: [Errno 61] Connection refused
https://demo.ilias.de,Demo,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: <HTTPError 302: 'The HTTP server returned a redirect error that would lead to an infinite loop.\nThe last 30x error message was:\nFound'> | reason: The HTTP server returned a redirect error that would lead to an infinite loop.
The last 30x error message was:
Found
https://restya.com/board/,Restyaboard,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: URLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1123)')) | reason: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1123)
https://demo.espocrm.com/,Demo,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: URLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1123)')) | reason: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1123)
https://git.ring.cx/savoirfairelinux/ring-project,Source Code,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: URLError(OSError(51, 'Network is unreachable')) | reason: [Errno 51] Network is unreachable
https://tiki.org/Demo,Demo,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: <HTTPError 301: 'The HTTP server returned a redirect error that would lead to an infinite loop.\nThe last 30x error message was:\nFound'> | reason: The HTTP server returned a redirect error that would lead to an infinite loop.
The last 30x error message was:
Found
https://demo.getsaleor.com/,Demo,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: URLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1123)')) | reason: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1123)
https://code.freedombone.net/bashrc/freedombone,Source Code,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: URLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate (_ssl.c:1123)')) | reason: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate (_ssl.c:1123)
https://titra.io/en/free-time-tracking-online/,Titra,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: <HTTPError 404: 'Not Found'> | reason: Not Found
https://wreeto.com,Wreeto,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: URLError(ConnectionRefusedError(61, 'Connection refused')) | reason: [Errno 61] Connection refused
https://dispatch.khlieng.com/connect,Demo,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: URLError(ConnectionRefusedError(61, 'Connection refused')) | reason: [Errno 61] Connection refused
https://magento.com/schedule-a-demo,Demo,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: timeout('The read operation timed out') | reason: 
https://restya.com/board/demo,Demo,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: URLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1123)')) | reason: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1123)
https://bbpress.trac.wordpress.org/browser/trunk,Source Code,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: <HTTPError 403: 'Forbidden'> | reason: Forbidden
https://piwigo.org/demo/,Demo,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: <HTTPError 404: 'Not Found'> | reason: Not Found
https://traduora.com,Traduora,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: URLError(ConnectionRefusedError(61, 'Connection refused')) | reason: [Errno 61] Connection refused
https://www.citadel.org/doku.php,Citadel,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: <HTTPError 404: 'Not Found'> | reason: Not Found
https://www.sakaiproject.org/try-sakai,Demo,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: <HTTPError 404: 'Not Found'> | reason: Not Found
https://hackershare.dev,Demo,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: timeout('The read operation timed out') | reason: 
https://s.wx0.de,Demo,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: <HTTPError 502: 'Bad Gateway'> | reason: Bad Gateway
https://apostrophecms.org/,Apostrophe,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: URLError(SSLCertVerificationError(1, "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'apostrophecms.org'. (_ssl.c:1123)")) | reason: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'apostrophecms.org'. (_ssl.c:1123)
https://jarr.info/login?next=%2F,Demo,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: <HTTPError 404: 'Not Found'> | reason: Not Found
https://friends-radio.herokuapp.com/,Demo,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: timeout('The read operation timed out') | reason: 
https://amiunique.org/,AmIUnique,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: URLError(ConnectionRefusedError(61, 'Connection refused')) | reason: [Errno 61] Connection refused
https://demo.getsiteinspector.com/reports,Demo,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: timeout('The read operation timed out') | reason: 
https://demo.funkwhale.audio/,Demo,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: <HTTPError 502: 'Bad Gateway'> | reason: Bad Gateway
https://cytu.be,Demo,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: <HTTPError 503: 'Service Temporarily Unavailable'> | reason: Service Temporarily Unavailable
https://code.eliotberriot.com/funkwhale/funkwhale,Source Code,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: URLError(gaierror(8, 'nodename nor servname provided, or not known')) | reason: [Errno 8] nodename nor servname provided, or not known
https://demo.openremote.io/,Demo,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: <HTTPError 308: 'Permanent Redirect'> | reason: Permanent Redirect
https://hg.dotclear.org/dotclear,Source Code,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: <HTTPError 503: 'Service Temporarily Unavailable'> | reason: Service Temporarily Unavailable
https://prologic.github.io/tube,Tube,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: <HTTPError 404: 'Not Found'> | reason: Not Found
https://demo.thingsboard.io/signup,Demo,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: <HTTPError 404: 'Not Found'> | reason: Not Found
https://v2.femto.pw/,Web-File-Uploader,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: URLError(gaierror(8, 'nodename nor servname provided, or not known')) | reason: [Errno 8] nodename nor servname provided, or not known
https://feedka.herokuapp.com,Demo,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: timeout('The read operation timed out') | reason: 
https://www.vimbadmin.net/demo/auth/login,Demo,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: <HTTPError 500: 'Internal Server Error'> | reason: Internal Server Error
https://demoshop.oxid-esales.com/community-edition/,Demo,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: <HTTPError 401: 'Unauthorized'> | reason: Unauthorized
https://demo.odoo.com/,Demo,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: <HTTPError 404: 'NOT FOUND'> | reason: NOT FOUND
https://tiki.org,Tiki,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: <HTTPError 301: 'The HTTP server returned a redirect error that would lead to an infinite loop.\nThe last 30x error message was:\nFound'> | reason: The HTTP server returned a redirect error that would lead to an infinite loop.
The last 30x error message was:
Found
https://getpinry.com/,Pinry,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: <HTTPError 308: 'Permanent Redirect'> | reason: Permanent Redirect
https://www.commafeed.com/,CommaFeed,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: timeout('The read operation timed out') | reason: 
https://tonken.glitch.me/,Demo,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: timeout('The read operation timed out') | reason: 
https://magento.com/,Magento,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: timeout('The read operation timed out') | reason: 
https://mydrive-demo.herokuapp.com/,Demo,https://github.com/awesome-selfhosted/awesome-selfhosted#readme,Self Hosted error: timeout('The read operation timed out') | reason: 